docs/loading/S3_Load.md

```diff
@@ -0,0 +1,272 @@
+
+```SQL
+LOAD LABEL user_behavior
+(
+    DATA INFILE("s3a://starrocks-datasets/user_behavior_sample_data.parquet")
+    INTO TABLE user_behavior
+    FORMAT AS "parquet"
+ )
+ WITH BROKER
+ (
+    "aws.s3.enable_ssl" = "true",
+    "aws.s3.use_instance_profile" = "false",
+    "aws.s3.region" = "us-west-1",
+    "aws.s3.access_key" = "AAAAAAAAAAAAAAAAAAAA",
+    "aws.s3.secret_key" = "BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"
+ )
+PROPERTIES
+(
+    "timeout" = "72000"
+);
+```
+
+#### Check progress
+
+Use the `SHOW LOAD` query to track progress. If you have multiple `LOAD` jobs running you can specify the `LABEL` associated with the job. In the output below there are two entries for the load job `user_behavior`. The first record shows a failure; scroll to the end of the table, and you see that access was forbidden. The second record succeeded with a valid AWS IAM access key and secret.
+
+```SQL
+SHOW LOAD;
+```
+
+```SQL
+SHOW LOAD WHERE label = 'user_behavior';
+```
+
+|JobId|Label|State|Progress|Type|Priority|ScanRows|FilteredRows|UnselectedRows|SinkRows|EtlInfo|TaskInfo|ErrorMsg|CreateTime|EtlStartTime|EtlFinishTime|LoadStartTime|LoadFinishTime|TrackingSQL|JobDetails|
+|----|------|-----|--------|----|--------|--------|------------|--------------|--------|-------|--------|--------|----------|------------|-------------|-------------|--------------|-----------|----------|
+|10104|user\_behavior|CANCELLED|ETL:N/A; LOAD:N/A|BROKER|NORMAL  |0       |0           |0             |0     |      |resource:N/A; timeout(s):72000; max\_filter\_ratio:0.0|type:ETL\_RUN\_FAIL; msg:unknown error when get file status: s3a://starrocks-datasets/user\_behavior\_sample\_data.parquet: getFileStatus on s3a://starrocks-datasets/user\_behavior\_sample\_data.parquet: com.amazonaws.services.s3.model.AmazonS3Exception: Forbidde|2023-08-04 15:58:51|                   |                   |                   |2023-08-04 15:58:57|           |{"All backends":{},"FileNumber":0,"FileSize":0,"InternalTableLoadBytes":0,"InternalTableLoadRows":0,"ScanBytes":0,"ScanRows":0,"TaskNumber":0,"Unfinished backends":{}} |
+|10110|user\_behavior|LOADING  |ETL:100%; LOAD:0%|BROKER|NORMAL  |0|0|0|0||resource:N/A; timeout(s):72000; max\_filter\_ratio:0.0||2023-08-04 16:22:27|2023-08-04 16:22:30|2023-08-04 16:22:30|2023-08-04 16:22:30|||{"All backends":{"563ec0b9-1bd6-4a39-aa30-1332fc0fee04":\[10004]},"FileNumber":1,"FileSize":1225637388,"InternalTableLoadBytes":0,"InternalTableLoadRows":0,"ScanBytes":0,"ScanRows":0,"TaskNumber":1,"Unfinished backends":{"563ec0b9-1bd6-4a39-aa30-1332fc0fee|
+
+## Using the `FILES()` table function
+
+### `FILES()` advantages
+
+- `FILES()` can infer the data types of the columns of the Parquet data and generate the schema for a StarRocks table. This provides the ability to query the file directly from S3 with a `SELECT` or to create the table structure in StarRocks as part of the data loading process.
```

Comment:
"-" in the beginning of the paragraph should be removed, 'cause there are no other items to comprise an unordered list with this paragraph.

Reply:
OK

Reply:
note to self: test with CSV file with a header and without and see if the schema is inferred for CSV also.  If so, remove the word `Parquet` from this sentence.